src/integrationTest/resources/db/testmigration/V1_6__drop_and_recreate_auth_table.sql

```diff
@@ -0,0 +1,19 @@
+DROP TABLE judicial_office_authorisation;
+DROP TABLE authorisation_type;
+
+CREATE TABLE judicial_office_authorisation(
+    judicial_office_auth_id bigint,
+    elinks_id varchar(256),
+    jurisdiction varchar(256),
+    ticket_id bigint,
```

Comment:
ticket bigint..why we need that.Ideally int should be suffient.

Reply:
its been decided by Shashank in JIRA